src/components/events/EventsList/EventCard.tsx

```diff
@@ -1,24 +1,26 @@
 import ClientImage from "@/components/ClientImage";
 import { LargeButton } from "@/components/LargeButton";
 import { P } from "@/components/typography/P";
+import { defaultImage } from "@/utils/default";
 import { EventItem } from "@/utils/types";
 import { BookOpenIcon } from "@heroicons/react/20/solid";
 import moment from "moment";
+import Link from "next/link";
 
 const EventCard = ({ event }: { event: EventItem }) => {
   return (
-    <div>
+    <Link href={`/events/${event._id}`}>
```

Comment:
I have created a method called `slugify` which you can use to slugify the event title